tests/static_lib/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    windows::build! {
+        Microsoft::Web::WebView2::Win32::*,
```

Comment:
Can we be more specific and just request `CompareBrowserVersions`?
```suggestion
        Microsoft::Web::WebView2::Win32::CompareBrowserVersions,
```